packages-ui/roosterjs-react/lib/ribbon/type/RibbonButtonStringKeys.ts

```diff
@@ -186,6 +186,11 @@ export type UndoButtonStringKey = 'buttonNameUndo';
  */
 export type CellShadeButtonStringKey = 'buttonNameCellShade' | BackgroundColorKeys;
 
+/**
+ * Key of localized strings of Spacing button
+ */
+export type SpacingButtonStringKey = 'buttonNameSpacing';
```

Comment:
If we don't add new ribbon button to roosterjs-react, no need to add string key here. You can hard code the string in demo code